### PR TITLE
Only add icedtea-web to jdk8 msi installer

### PR DESCRIPTION
**Pull request by lumpfish**

### CODE REVIEW

**Comment by lumpfish** [comment]
@douph1 , @gdams - is there some way I can test his change using the AdoptOpenJDK build machines with the Wix setup?

File: wix/Build.OpenJDK_generic.cmd [hunk 1]
```diff
@@ -178,7 +178,7 @@ FOR %%A IN (%ARCH%) DO (
 
 	REM Build without extra Source Code feature
 
-    IF !PLATFORM! == x64 (
+    IF !PLATFORM! == x64 IF !PRODUCT_MAJOR_VERSION! == 8 (
```
Comment (karianna) [inline wix/Build.OpenJDK_generic.cmd]: I would nest that 2nd IF to match the style of the rest of this file

Comment (lumpfish) [reply wix/Build.OpenJDK_generic.cmd]: Done

**Comment by lumpfish** [comment]
I've changed the compound IF to a nested IF as per @karianna 's comment.

**Pull request merged**